automation/vars/shiftonstack.yaml

```diff
@@ -0,0 +1,100 @@
+---
+# Stores the different automation workflow steps required by various
+# shiftonstack deployed topologies.
+vas:
+  shiftonstack03:
```

Comment:
would you mind to rename to `shiftstack` ?

Reply:
Done